install.ps1

```diff
@@ -1,84 +1,86 @@
 $zipSavePath = "$themePath\bloom-main.zip"
-Write-Host "Downloading spicetify-bloom latest master..."
+Write-Host -Object "Downloading spicetify-bloom latest master..."
 Invoke-WebRequest -Uri $zipUri -UseBasicParsing -OutFile $zipSavePath
 
 # Extract theme from .zip file
-Write-Host "Extracting..."
+Write-Host -Object "Extracting..."
 Expand-Archive -Path $zipSavePath -DestinationPath $themePath -Force
-Get-ChildItem "$themePath\spicetify-bloom-main\*" | ForEach-Object { Move-Item $_ $themePath }
-Remove-Item "$themePath\spicetify-bloom-main"
+Move-Item -Path "$themePath\spicetify-bloom-main\*" -Destination $themePath
+Remove-Item -Path "$themePath\spicetify-bloom-main"
 
 # Delete .zip file
-Write-Host "Deleting zip file..."
+Write-Host -Object "Deleting zip file..."
 Remove-Item -Path $zipSavePath
 
-# Change Directory to the Theme Folder
-Set-Location $themePath
-
 # Copy the bloom.js to the Extensions folder
-Copy-Item bloom.js ..\..\Extensions
-Write-Host "+ Installed bloom.js theme"
+Copy-Item -Path "$themePath\bloom.js" -Destination "$spicePath\Extensions"
+Write-Host -Object "Installed bloom.js theme"
 
 # Apply the theme with spicetify config calls
 spicetify config extensions bloom.js
 spicetify config current_theme bloom
 spicetify config color_scheme dark
 spicetify config inject_css 1 replace_colors 1 overwrite_assets 1
-Write-Host "+ Configured bloom theme"
+Write-Host -Object "Configured bloom theme"
 
 # Patch the xpui.js for sidebar fixes
 # credit: https://github.com/JulienMaille/dribbblish-dynamic-theme/blob/main/install.ps1
-$configFile = Get-Content "$spicePath\config-xpui.ini"
-if (-not ($configFile -match "xpui.js_find_8008")) {
+$configFile = Get-Content -Path "$spicePath\config-xpui.ini"
+if ($configFile -notmatch "xpui.js_find_8008") {
   $rep = @"
 [Patch]
-xpui.js_find_8008=,(\w+=)32,
-xpui.js_repl_8008=,`${1}58,
+xpui.js_find_8008 = , (\w+=)32,
+xpui.js_repl_8008 = , `${1}58,
 "@
   # In case missing Patch section
-  if (-not ($configFile -match "\[Patch\]")) {
+  if ($configFile -notmatch "\[Patch\]") {
     $configFile += "`n[Patch]`n"
   }
-  $configFile = $configFile -replace "\[Patch\]",$rep
-  Set-Content "$spicePath\config-xpui.ini" $configFile
+  $configFile = $configFile -replace "\[Patch\]", $rep
+  Set-Content -Path "$spicePath\config-xpui.ini" -Value $configFile
 }
-Write-Host "+ Patched xpui.js for Sidebar fixes"
+Write-Host -Object "Patched xpui.js for Sidebar fixes"
 
 # backup apply or just apply where necessary
-# credit: https://github.com/JulienMaille/dribbblish-dynamic-theme/blob/main/install.ps1
-$backupVer = $configFile -match "^version"
-$version = ConvertFrom-StringData $backupVer[0]
-if ($version.version.Length -gt 0) {
+$configFile | ForEach-Object {
+  if ($_ -match "^version = (.+)$") {
+    $version = $matches[1]
+  }
+}
+
+if ($version) {
   spicetify apply
-} else {
+}
+else {
   spicetify backup apply
 }
-Write-Host "+ Applied Theme"
+Write-Host -Object "Applied Theme"
```

Comment:
Keep the newline. Even though it doesn't have effect on Windows users or usability of the script, it does have effect on the log. We don't want changes to be pointed to wrong commit.

Reply:
should I make commit adding newline?

Reply:
Just amend to the commit removing the newline.